### DISCUSSION TITLE
add support for autorun

### DISCUSSION THREAD

**Discussion by superiums** [YaLTeR/niri#155] [Ideas]
althrough niri already support xdg-autostart,
but i'm in void linux which doesn't have systemd.

it's more flex to add a autostart script. just like dwl/sway/river do.
these script must be launch after wm, such as fcitx, waybar ..., so they can't be added to niri-session script.

**Comment by YaLTeR** [comment]
```
// Add lines like this to spawn processes at startup.
// Note that running niri as a session supports xdg-desktop-autostart,
// which may be more convenient to use.
// spawn-at-startup "alacritty" "-e" "fish"
```
